### PR TITLE
Update beats framework to 4702e7a

### PR DESCRIPTION
**Pull request by graphaelli**
Pull in docker image fixes from elastic/beats#15069

### CODE REVIEW

File: _beats/libbeat/scripts/Makefile [hunk 1]
```diff
@@ -51,6 +51,7 @@ BEAT_KIBANA_PASS?=${KIBANA_ES_PASS}
 # Define a superuser, e.g. for initial setup
 ES_SUPERUSER_USER?=admin
 ES_SUPERUSER_PASS?=changeme
+>>>>>>> master
```
Comment (axw) [inline _beats/libbeat/scripts/Makefile]:
```suggestion
```

Comment (graphaelli) [reply _beats/libbeat/scripts/Makefile]: ouch nice catch

**Comment by graphaelli** [comment]
jenkins, run the tests please

**Comment by graphaelli** [comment]
redoing this

**Pull request closed without merging**